docs/sources/http.md

```diff
@@ -0,0 +1,52 @@
+# Event source for HTTP
+
+This event source exposes a generic HTTP endpoint that turns user requests into [CloudEvents][ce] to be consumed by other Triggermesh components. This source can fit integration scenarios where a webhook that might receive events from an external service is required.
+
+## Prerequisites
+
+An external client that executes HTTP requests.
+When using basic auth, a secret containing the password is required.
+
+## Create the HTTP source integration
```

Comment:
Nitpicky -I hate to do this one haha. To follow the MLA guidelines for titles, `Create the HTTP source integration `here should be capitalized as` Create the HTTP Source Integration`